examples/base/kustomization.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
base -> generic wdyt?

base maybe be interpreted like do this first

I am ok with not calling it either and keeping this directly under examples directory as well

Reply:
sure